config/webpack-config.js

```diff
@@ -27,13 +27,11 @@ module.exports = {
   plugins: [
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME ),
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME,
-                                             [ COMMON_BUNDLE_NAME ] ),
+                                             'common.[chunkhash].js' ),
```

Comment:
> So how's this working now? It generates common.js and common.*.js that are duplicates of each other, is that right? Can we just spit out the hashed version?

So it's only generating the hashed version. The problem is the directory doesn't get cleaned. If you `gulp clean && gulp build`, then you should only see the hashed version.